executor/runtime/docker/docker.go

```diff
@@ -1873,6 +1873,34 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(v1Container v1.Container, ma
 	return dockerContainerConfig, dockerHostConfig, dockerNetworkConfig, nil
 }
 
+// computeExtraContainersDockerEntrypoint takes an extra container, and tries our best to take multiple
+// inputs, and computing the most sane entrypoint we can come up with given our requirements. That is:
+// 1. We have input k8s container "Command" (docker entrypoint)
+// 2. We have the original docker entrypoint on the image
+// 3. We have tini, which we need to inject ourselves because we want tini goodness (stdout/err, seccomp, etc)
+//
+// Given those 3 things, we must return *something* that docker can actually run.
+func computeExtraContainersDockerEntrypoint(c *runtimeTypes.ExtraContainer) []string {
+	// What docker calls "entrypoint", k8s calls "command", but in addition, we prepend tini
+	// The reason we do this is because, even with init=true, docker will only inject tini
+	// on containers running in a private pid namespace.
+	// On titus, we want tini on *every* container, because it gives us features like stdout/err
+	originalEntrypoint := getExtraContainerEntrypoint(c)
+	return append([]string{"/sbin/docker-init", "-s", "--"}, originalEntrypoint...)
+}
+
+// getExtraContainerEntrypoint computes the original entrypoint that the user
+// wants to run, given two sources:
+// 1. the input k8s pod "command" (takes precidence)
+// 2. the input ENTRYPOINT on the image (should be used if no command is specified on the container spec)
```

Comment:
Would it make sense to fall back to the COMMAND on the image, or is that redundant with the value from V1Container?

Reply:
I don't think so.
I'm pretty sure docker will do that for us.